### DISCUSSION TITLE
How to trigger the toast I created

### DISCUSSION THREAD

**Discussion by wodoame** [wodoame/expense-tracking-application#11] [Show and tell]
Okay so let me demonstrate how you can trigger the toast.

**Comment by Grandkojo** [comment]
Alright, let's discuss

**Reply by wodoame** [reply]
Okay so there's a html snippet called `toastWrapper.html` you'll have to add that to the particular file where you want the toast to appear
so you'll do

```html
{ % include  'core/components/toastWrapper.html' % }
```

**Reply by wodoame** [reply]
You have to link a js file to make it work. 
```html 
     <script src="{% static 'js/toast.js' %}"></script>
```

**Reply by Grandkojo** [reply]
okay, would do this shortly

**Reply by Grandkojo** [reply]
Done adding the two files

**Reply by wodoame** [reply]
Then after doing this you have to set a message (e.g `messages.success(request, 'your message' )`) in a view before rendering the page that you want the toast to appear.

**Reply by Grandkojo** [reply]
Oh okay, let me try it out in a bit :)

**Reply by wodoame** [reply]
Add this script  to make the close button work
```html 
    <script src="{% static 'js/flowbitejs.min.js' %}"></script>
```

**Reply by Grandkojo** [reply]
yeah, i was  about to ask about this

**Reply by Grandkojo** [reply]
Alright, it's working perfectly now :)
